.circleci/config.yml

```diff
@@ -11,34 +11,29 @@ jobs:
       - run:
           name: Download Code Climate test-reporter
           command: |
-            mkdir tmp
-            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-0.6.3-linux-amd64 > tmp/cc-test-reporter
-            chmod +x tmp/cc-test-reporter
+            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-0.6.3-linux-amd64 > /tmp/cc-test-reporter
+            chmod +x /tmp/cc-test-reporter
 
       - persist_to_workspace:
-          root: tmp
+          root: /tmp
           paths:
             - cc-test-reporter
 
   coverage-upload:
     docker:
-      - image: decidim/decidim@sha256:c0588c1f425fa2433733978fbfa147b4e6cc54be9f6ac4583920158d6a408972
+      - image: circleci/ruby:2.5
```

Comment:
Just to know, did the decidim image work? I guess that no, but just to confirm

Reply:
Yeah, it would work, actually. However, you don't need a big image here (decidim's image is more that 1Gb last time I checked). Actually, I wonder if https://hub.docker.com/_/scratch/ would work here, since the only thing this job does is using a binary and some artifacts provided by other jobs.